options/options.c

```diff
@@ -114,6 +114,10 @@ static const m_option_t mp_vo_opt_list[] = {
     {"ontop-level", OPT_CHOICE(ontop_level, {"window", -1}, {"system", -2},
         {"desktop", -3}), M_RANGE(0, INT_MAX)},
     {"border", OPT_BOOL(border)},
+#if HAVE_WIN32_DESKTOP
+    {"window-affinity", OPT_CHOICE(window_affinity, {"default", WDA_NONE},
+        {"excludefromcapture", WDA_EXCLUDEFROMCAPTURE}, {"monitor", WDA_MONITOR})},
+#endif
```

Comment:
Seems I missed this the last time around, but you can move this one down where `{"vo-mmcss-profile", OPT_STRING(mmcss_profile)},` since it has the same guard.

Reply:
After this, the other pr will also have to be moved in between the guard, but 1 of them has to be merged first I guess.